chess/__init__.py

```diff
@@ -1221,14 +1221,19 @@ def __str__(self) -> str:
 
         return "".join(builder)
 
-    def unicode(self, *, invert_color: bool = False, borders: bool = False, empty_square: str = "⭘") -> str:
+    def unicode(self, *, invert_color: bool = False, borders: bool = False, empty_square: str = "⭘", flipped:bool = False) -> str:
         """
         Returns a string representation of the board with Unicode pieces.
         Useful for pretty-printing to a terminal.
 
         :param invert_color: Invert color of the Unicode pieces.
         :param borders: Show borders and a coordinate margin.
         """
+        board = self.copy()
+        if flipped:
+          board.apply_transform(flip_vertical)
+          board.apply_transform(flip_horizontal)
+
```

Comment:
This transformation will not change the coordinates if `board.unicode(..., borders=True)`. Best to change the actual iteration order instead.